syntax = "proto3";

package deepos_app;

service ApplicationStats {
  rpc SendAppStat(AppStat) returns (AppStatReply) {}
}

message AppStat {
  string stat_name = 1;
  float stat = 2;
}

message AppStatReply {
  string message = 1;
}
